synapse/rest/client/login.py

```diff
@@ -598,3 +599,12 @@ def register_servlets(hs, http_server):
     SsoRedirectServlet(hs).register(http_server)
     if hs.config.cas_enabled:
         CasTicketServlet(hs).register(http_server)
+
+
+def _load_sso_handlers(hs: "HomeServer"):
```

Comment:
Can you add a quick docstring, mainly its useful to know this is safe to call multiple times.